pkg/cli/tools/binary_tools.go

```diff
@@ -123,58 +119,41 @@ func GetValidPlatform(currentOS, currentArch string) (string, error) {
 	return platform, nil
 }
 
-// DownloadToFolder creates a folder and a file, uses the ORAS client to copy from the remote repository to the file,
-// and makes the file executable by everyone. An error is returned if any of these steps fail.
+// DownloadToFolder creates a folder and a file, downloads the bicep binary to the file,
 func DownloadToFolder(filepath string) error {
-	// create folders
-	err := os.MkdirAll(path.Dir(filepath), os.ModePerm)
-	if err != nil {
-		return fmt.Errorf("failed to create folder %s: %v", path.Dir(filepath), err)
-	}
-
-	// Create a file store
-	fs, err := file.New(path.Dir(filepath))
-	if err != nil {
-		return fmt.Errorf("failed to create file store %s: %v", filepath, err)
-	}
-	defer fs.Close()
-
-	ctx := context.Background()
-	platform, err := GetValidPlatform(runtime.GOOS, runtime.GOARCH)
+	// Create the file
+	bicepBinary, err := os.Create(filepath)
 	if err != nil {
 		return err
 	}
+	defer bicepBinary.Close()
 
-	// Define remote repository
-	repo, err := remote.NewRepository(binaryRepo + platform)
+	// Get file extension
+	extension, err := getFileExtension()
 	if err != nil {
 		return err
 	}
 
-	// Copy the artifact from the registry into the file store
-	tag := version.Channel()
-	if version.IsEdgeChannel() {
-		tag = "latest"
-	}
-	_, err = oras.Copy(ctx, repo, tag, fs, tag, oras.DefaultCopyOptions)
+	// Get the data
+	resp, err := http.Get(binaryRepo + extension)
 	if err != nil {
 		return err
 	}
+	defer resp.Body.Close()
 
-	// Open the folder so we can mark it as executable
-	bicepBinary, err := os.Open(filepath)
+	// Write the body to file
+	_, err = io.Copy(bicepBinary, resp.Body)
```

Comment:
What would happen if...

- There's an existing version of bicep on disk.
- The existing copy on disk is larger than the new version.

Reply:
This should only download the binary to .rad directory. So if there's an existing version of that file, it will get overridden but if there's another bicep binary somewhere else, I don't believe that will get updated. 

The size of the updated file would be that of the new version. os.Create() truncates the file if it exists so it should be the size of the new file after io.Copy()

Let me know if any of that seems wrong